app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,23 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    # @param [Hash] options from blacklight invocation of helper_method
+    # @return the html_safe facet links separated by " > " to indicate facet hierarchy
+
+    def link_to_each_facet_field(options)
+      raise ArgumentError unless options[:config] && options[:value] && options[:config][:helper_facet]
```

Comment:
As part of the ArgumentError, can you add the expectations of the options? (e.g. "Expected options to have keys :value and :config. Expected options[:config] to have key :helper_facet")